### DISCUSSION TITLE
Matrix Awtrix Pro und Buttons Probleme | Matrix Awtrix Pro and Button Issues

### DISCUSSION THREAD

**Discussion by 3DGera** [pixelit-project/PixelIt#361] [Help]
Hallo,

ich habe eine alte Awtrix Pro Matrix und möchte nun von Awtrix zu Pixellt wechseln, aber die Matrix wird leider nicht angesteuert. Ich habe bereits alle Matrix-Typen getestet, jedoch ohne Erfolg.

Das zweite Problem, das ich habe, ist, dass die Buttons rechts und links nicht funktionieren. Ich habe die GPIOs angepasst, da ich früher die Awtrix mit einem normalen Wemos D1 genutzt habe, aber jetzt funktioniert nur der mittlere Button.

Angeschlossen sind die Buttons wie in der Dokumentation beschrieben, also:

Links: D0 - GPIO26 - io26
Mitte: D4 - GPIO16 - io16
Rechts: D8 - GPIO5 - io5
Den GPIO5 kann ich in den Optionen jedoch gar nicht auswählen.

Zur Findung der GPIOs habe ich folgende Bilder genutzt.

----

Hello,

I have an old Awtrix Pro Matrix and now want to switch from Awtrix to Pixellt, but unfortunately, the matrix is not being controlled. I have already tested all the matrix types without success.

The second problem I have is that the buttons on the left and right are not working. I adjusted the GPIOs since I previously used Awtrix with a regular Wemos D1, but now only the middle button works.

The buttons are connected as described in the documentation:

Left: D0 - GPIO26 - io26
Middle: D4 - GPIO16 - io16
Right: D8 - GPIO5 - io5

However, I can't select GPIO5 in the options.

I used the following images to identify the GPIOs.

![original](https://github.com/user-attachments/assets/facdaabb-0d17-4f23-b68b-ca12b7d6dcd6)
![image](https://github.com/user-attachments/assets/a29d0cda-d487-48e3-8749-fc5a63e0c1a1)

**Comment by foorschtbar** [comment]
Hi, hast du mal ein paar Bilder von dem ESP und der verkabelung? Grundsätzlich sollte das funktionieren, wir haben viele Wechsler. Normal reicht ein Update der Firmware, ggf. vorher löschen und den Pin für die LEDs ggf. umlöten. Der rest kann über die UI konfiguriert werden.

**Comment by 3DGera** [comment]
Hallo,

ich nutze auch das PCB von Awtrix =) ich habe aber alles kontrolliert und die Pinbelegung passt auch zu euren.

![IMG_3943](https://github.com/user-attachments/assets/97dc321a-4afd-4d84-b48a-9e60835fd31d)

Ich benutze momentan die Awtrix 3 Firmware, dort muss man damit die Matrix funktioniert eine Datei (dev.json) anlegen in der man die Matrix angeben muss die man nutzt. Bei mir wäre es dann die 2^

Leider gibt's zu der PCB keine Dokumentation mehr :(

**Comment by 3DGera** [comment]
Ich hab's jetzt gerade noch mal mit einem ESP8266 und hier funktioniert alles sofort :D warum es mit dem ESP32 nicht funktioniert weiß ich nicht.
